README.md

```diff
@@ -62,7 +62,7 @@ This image respects the following custom environment variables to be passed duri
 - `build_time_zones`: `True` builds the timezone db which is needed for time-dependent routing. `Force` will do the same, but first delete the existing db. Default `False`.
 - `build_transit`: `True` will attempt to build transit tiles if none exist yet. `Force` will remove existing transit **and** routing tiles. Default `False`.
 - `build_tar` (since 29.10.2021/v`3.1.5`): `True` creates a tarball of the tiles including an index which allows for extremely faster graph loading after reboots. `Force` will do the same, but first delete the existing tarball. Default `True`.
-- `server_threads`: How many threads `valhalla_build_tiles` will use and `valhalla_service` will run with. Default is the value of `nproc`.
+- `server_threads`: How many threads `valhalla_build_tiles` will use and `valhalla_service` will run with. Default is the value of `nproc`. If valhalla kills it self when building transit, lower this number. 
```

Comment:
This is a general advice IMO, also without transit it’ll be killed when building the graph. I’d suggest to remove „transit“

Reply:
Good point. I added this in another commit to "building tiles" instead of "building transit"